### PR TITLE
Included singletons from AdapterRemoval in the flash fastq out file.

### LINKED ISSUE #1: Add reads from singletons file from AdapterRemoval to flash output

### PR DESCRIPTION
**Pull request by TaliVeith**
Fixes #1

### CODE REVIEW

File: 005-trim/trim.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ fastq2=$dataDir/$task2.fastq.gz
 log=$logDir/$task.log
 out=$task.trim.fastq.gz
 out2=$task2.trim.fastq.gz
+out3=$task.singleton.fastq.gz
```
Comment (terrycojones) [inline 005-trim/trim.sh]: It would be better to call this `singletons` as you do in the flash script.

File: 007-flash/flash.sh [hunk 2]
```diff
@@ -59,6 +60,11 @@ function doFlash()
 
     echo "  Compressing combined FASTQ into $out at $(date)." >> $log
     gzip $outUncompressed
+
+    if [ -f $singletons ]
```
Comment (terrycojones) [inline 007-flash/flash.sh:66]: I would add a comment before this to say what's going on and why the file may not be present.

**Comment by terrycojones** [comment]
I just made a couple of tiny comments re making things a bit clearer. Looks good!

When you merge it, you can go into `~/rdsbt/projects/charite/csd3-pipeline` and in there do a `git pull origin master` and the changes should then be pulled down onto the server.  That won't put them in place in the various data dirs, though. We can talk about whether/which dirs to do that in.

File: 005-trim/trim.sh (hunk 1 above)
Comment (TaliVeith) [reply 005-trim/trim.sh]: done

File: 007-flash/flash.sh (hunk 2 above)
Comment (TaliVeith) [reply 007-flash/flash.sh:66]: done

**Comment by TaliVeith** [comment]
Thanks!
Can you merge the pull request or give me write access? I can't merge it atm.

**Comment by terrycojones** [comment]
I changed the perms so you have write permission.

I'd change `out3` in `005-trim/trim.sh` to be called `singletons` too (the `out` and `out2` variables in that script are also not much good, but no need to add another one of a different file type but another `out` name. This is a tiny issue in this code base because it's so small, but this is still a "teachable moment"!).

**Pull request merged**